chips/opentitan/chip.toml

```diff
@@ -0,0 +1,29 @@
+[gpio0]
+address = 0x4004_0000
+size = 0x1000
+
+[uart0]
+address = 0x4000_0000
+size = 0x1000
+
+[aontimer]
+address = 0x4047_0000
+size = 0x1000
+interrupts = { bark = 158 }
+
+[plic]
+address = 0x4800_0000
+size = 0x0400_4004 
+interrupts = { irq = 11 }
+
+[timer]
+address = 0x4010_0000
+size = 0x1000
+
+[mtime]
+address = 0x4010_0110
+size = 0
+
+[mtimecmp]
+address = 0x4010_0118
+size = 0
```

Comment:
The address of `mtime` and `mtimecmp` in QEMU right now uses the old addresses of the Ibex timer. Here's the old addresses:
- mtime: `0x4010_0104`
- mtimecmp: `0x4010_10C`

QEMU uses commit 217a0168ba118503c166a9587819e3811eeb0c0c for its OT implementation.

In the future when QEMU updates its devices, the addresses we have now will be correct.

Reply:
If correct means the latest addresses in the Ibex timer spec, then yes.